### PR TITLE
EVG-2659: make sure repotracker jobs run

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

**Comment by tychoish** [comment]
This is the thing that's running in staging

File: rest/data/repotracker.go [hunk 1]
```diff
@@ -31,6 +34,16 @@ func (c *RepoTrackerConnector) TriggerRepotracker(q amboy.Queue, msgID string, e
 	if len(branch) == 0 {
 		return nil
 	}
+	adminSettings, err := evergreen.GetConfig()
+	if err != nil {
+		return errors.Wrap(err, "error retrieving admin settings")
+	}
+	if adminSettings.ServiceFlags.RepotrackerDisabled {
+		grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
+			"message": "github push events triggering repotracker is disabled",
```
Comment (richardsamuels) [inline rest/data/repotracker.go]: Change message to "repotracker is disabled". Change the error just below this too.
Add another field: "source": "github hooks"

File: units/repotracker.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func (j *repotrackerJob) Run() {
 		j.AddError(errors.Wrap(err, "error retrieving admin settings"))
 		return
 	}
-	if adminSettings.ServiceFlags.RepotrackerPushEventDisabled {
+	if adminSettings.ServiceFlags.RepotrackerDisabled {
 		grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
 			"job":     repotrackerJobName,
 			"message": "github push events triggering repotracker is disabled",
```
Comment (richardsamuels) [inline units/repotracker.go]: Change message/error below to read "repotracker is disabled"

File: units/repotracker.go [hunk 3]
```diff
@@ -67,7 +67,7 @@ func (j *repotrackerJob) Run() {
 		j.AddError(errors.Wrap(err, "error retrieving admin settings"))
 		return
 	}
-	if adminSettings.ServiceFlags.RepotrackerPushEventDisabled {
+	if adminSettings.ServiceFlags.RepotrackerDisabled {
 		grip.InfoWhen(sometimes.Percent(evergreen.DegradedLoggingPercent), message.Fields{
 			"job":     repotrackerJobName,
```
Comment (richardsamuels) [inline units/repotracker.go]: Can you add "job_id": j.ID() to the Fields object please?

**Pull request merged**